### ISSUE TITLE
webapp's npm install fails

### ISSUE THREAD

**Issue by nils-wisiol** [desec-io/desec-stack#517]
If there is no package-lock.json present and npm version is 7.4.3, `npm install` fails.

Providing a package-lock.json file or using npm 6.14.11 solves (or circumvents) the problem.

In the worst case, this could break the webapp during deployment.

Here is a backup package-lock.json that enables npm install with npm version 7.4.3. 
[package-lock.json.gz](https://github.com/desec-io/desec-stack/files/5994945/package-lock.json.gz)

**Comment by unuseless** [comment]
It's good practice to check in the package-lock.json into git.  This allows to install with `npm ci`, and guarantees you get the same packages as e.g. were used for testing.
So, please provide a package-lock.json.

**Comment by nils-wisiol** [comment]
@unuseless, would you be able to provide a PR for this? I'm unfamiliar with the details of npm best practice, your help would be most welcome

**Comment by unuseless** [comment]
@nils-wisiol At the moment I'm unable to provide a PR, but I can give you the commands here:
```
# to make sure you have a recent package-lock.json
npm install
# check if your packages are updated
npm outdated
# maybe update
npm update
# add package-lock.json to git, maybe remove from .gitignore?
git add .
git commit -m "added package-lock.json"
# edit README
```
Add to the README: To install the dependencies for the webapp, use `npm ci`.

**Comment by peterthomassen** [comment]
fixed in #611
